corehq/apps/translations/models.py

```diff
@@ -168,6 +168,10 @@ def __str__(self):
     def plaintext_api_token(self):
         return b64_aes_decrypt(self.api_token)
 
+    @plaintext_api_token.setter
+    def plaintext_api_token(self, plaintext):
```

Comment:
I got a bit confused here. Shouldn't it be setter to `api_token`, instead of `plaintext_api_token`?

Reply:
I see. Just read a bit about setter to refresh my memory! Thank you for explanation!